app/controllers/stories_controller.rb

```diff
@@ -330,4 +331,58 @@ def assign_podcasts
   def assign_classified_listings
     @classified_listings = ClassifiedListing.where(published: true).select(:title, :category, :slug, :bumped_at)
   end
+
+  def set_json_ld
+    @user_json_ld = {
+      "@context": "http://schema.org",
+      "@type": "Person",
+      "mainEntityOfPage": {
+        "@type": "WebPage",
+        "@id": "https://dev.to/<%= user_url(@user) %>"
+      },
+      "url": "<%= user_url(@user) %>",
+      "sameAs": [],
+      "image": "<%= ProfileImage.new(@user).get(width: 320) %>",
+      "name": "<%= @user.name %>",
+      "email": "",
+      "jobTitle": "",
+      "description": "<%= @user.summary.presence || ['404 bio not found'].sample %>",
+      "disambiguatingDescription": [],
+      "worksFor": [
+        {
+          "@type": "Organization"
+        },
+      ],
+      "alumniOf": ""
+    }
+    set_user_profile_json_ld
+    set_user_same_as_json_ld
+  end
+
+  def set_user_profile_json_ld
+    @user_json_ld["disambiguatingDescription"]&.append(@user.mostly_work_with) if @user.mostly_work_with.present?
```

Comment:
you can remove the `&` as you already know the key exists and it's an array. Same for the other similar keys